core/src/main/scala/org/apache/spark/internal/io/FileCommitProtocol.scala

```diff
@@ -231,12 +240,3 @@ object FileCommitProtocol extends Logging {
     new Path(path, ".spark-staging-" + jobId)
   }
 }
-
-/**
- * The specification for Spark output file name.
- * This is used by [[FileCommitProtocol]] to create full path of file.
- *
- * @param prefix Prefix of file.
- * @param suffix Suffix of file.
- */
-final case class FileNameSpec(prefix: String, suffix: String)
```

Comment:
isn't this a public class?

Reply:
I think because the companion object is defined here, so it cannot be imported in other files.

Reply:
Actually wait a second, let me double check. Sorry for the rush.